application-admintools-ui/src/main/resources/AdminTools/Code/AdminToolsJS.xml

```diff
@@ -191,7 +191,8 @@
   $(document).ready(function() {
     if ($('#healthCheck').length) {
       initialiseJobJS();
-      let jobState = $('.healthCheckJobState').attr('data-jobState');
+      let jobState = $('.health-check-job-state').attr('data-jobState');
```

Comment:
BTW, ``data-*`` attributes should use the kebab-case, see https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes . This way for an attribute like ``data-job-state`` you can write ``someElement.dataset.jobState``.